src/javascript/app/pages/user/account/settings/financial_assessment.js

```diff
@@ -45,6 +45,11 @@ const FinancialAssessment = (() => {
             });
         }
 
+        const is_professional = State.getResponse('get_account_status.professional') || State.getResponse('get_account_status.professional_requested');
+        if (is_professional) {
+            $('#trading_experience_form').setVisibility(0);
```

Comment:
this can come outside of the if condition and change to `$('#trading_experience_form').setVisibility(! is_professional);`